### PR TITLE
Lazily port auth.py to Python 3

### PR DESCRIPTION
**Pull request by f8l**
Done in accordance to [GI porting guide](https://wiki.gnome.org/Projects/PyGObject/IntrospectionPorting).

### CODE REVIEW

File: examples/data/scripts/auth.py [hunk 1]
```diff
@@ -3,6 +3,13 @@
 from __future__ import print_function
 
 import os
+try:
+    import pygtkcompat
+except ImportError:
+    pygtkcompat = None
+if not (pygtkcompat is None):
```
Comment (mathstuf) [inline examples/data/scripts/auth.py]: The more Pythonic way is `pygtkcompat is not None`.

Comment (f8l) [inline examples/data/scripts/auth.py]: You got me. This is where I deviated from the guide. I already forgot ‘is not’ is not ‘is’ ‘not’ in Python.
Don’t you think the less Pythonic way is more readable for those who don’t know ‘is not’ is parsed as a single operator?
(Yes, I know that’s a weak argument. Python code is supposed to be read by those who know Python, isn’t it?)

**Comment by mathstuf** [comment]
There are lots of things I find odd in Python :) . Either way, readability is subjective versus idiomatic being at least closer to objective (in a hive-mind kind of way).

**Comment by keis** [comment]
Yes, agreed, sticking to the conventions of python is better.

**Pull request merged**